.github/workflows/rocm.yml

```diff
@@ -25,12 +25,12 @@ jobs:
       sync-tag: rocm-build
       test-matrix: |
         { include: [
-          { config: "default", shard: 1, num_shards: 6, runner: "linux.rocm.gpu" },
-          { config: "default", shard: 2, num_shards: 6, runner: "linux.rocm.gpu" },
-          { config: "default", shard: 3, num_shards: 6, runner: "linux.rocm.gpu" },
-          { config: "default", shard: 4, num_shards: 6, runner: "linux.rocm.gpu" },
-          { config: "default", shard: 5, num_shards: 6, runner: "linux.rocm.gpu" },
-          { config: "default", shard: 6, num_shards: 6, runner: "linux.rocm.gpu" },
+          { config: "default", shard: 1, num_shards: 6, runner: "linux.rocm.gpu.2" },
+          { config: "default", shard: 2, num_shards: 6, runner: "linux.rocm.gpu.2" },
+          { config: "default", shard: 3, num_shards: 6, runner: "linux.rocm.gpu.2" },
+          { config: "default", shard: 4, num_shards: 6, runner: "linux.rocm.gpu.2" },
+          { config: "default", shard: 5, num_shards: 6, runner: "linux.rocm.gpu.2" },
+          { config: "default", shard: 6, num_shards: 6, runner: "linux.rocm.gpu.2" },
```

Comment:
From the PR description, it looks like this change should be reverted, shouldn't it?

Reply:
It will be, yes.  IIUC we've confirmed the MI210s have given a green signal now, so we will revert this change.